### ISSUE TITLE
Landing page CSS layout problem with portrait screen

### ISSUE THREAD

**Issue by huan** [wechaty/docusaurus#1381]
It seems that the render css has some issues when the screen is portraint:

![image](https://user-images.githubusercontent.com/1361891/138958100-05d12b0a-fa11-437c-b2bf-b109ae380f38.png)

**Comment by franknkurunziza** [comment]
can i work on this issue?

**Comment by huan** [comment]
Of course, I'd like to accept PRs for either fixing the problem on our website or improving it!

**Comment by franknkurunziza** [comment]
I'm Sorry my first time Contributing What do you mean by PRs?
